src/services/ledger/schema.ts

```diff
@@ -69,6 +69,24 @@ const transactionSchema = new Schema({
     default: 0,
   },
 
+  satsAmount: Number,
+  centsAmount: Number,
+  satsFee: {
+    type: Number,
+  },
+  centsFee: {
+    type: Number,
+  },
+
+  displayAmount: Number,
+  displayFee: {
+    type: Number,
+  },
+  displayCurrency: {
+    type: String,
+    enum: ["USD", "BTC"],
```

Comment:
please dont use enum, there are more than 150 display currencies

Reply:
Changed to string: https://github.com/GaloyMoney/galoy/pull/1269/commits/4f2636ccad3834e44bbb80239f8466b9c0c5d26e